test/acquisition/multi_objective/test_monte_carlo.py

```diff
@@ -1169,6 +1234,37 @@ def test_constrained_q_noisy_expected_hypervolume_improvement(self):
             mm._posterior._samples = samples
             res = acqf(X)
             self.assertAlmostEqual(res.item(), 1.5, places=4)
+            # test multiple constraints one eta with
+            # this crashes for large etas, and I do not why
+            # set the MockPosterior to use samples over baseline points
+            etas = [torch.tensor([1.0]), torch.tensor([1.0, 10.0])]
+            constraints = [
+                [lambda Z: torch.ones_like(Z[..., -1])],
+                [
+                    lambda Z: torch.ones_like(Z[..., -1]),
+                    lambda Z: torch.ones_like(Z[..., -1]),
+                ],
+            ]
```

Comment:
Could make this a bit cleaner, but not really necessary
```suggestion
            
            def con_func(Z):
                torch.ones_like(Z[..., -1])
            
            constraints = [[con_func], [con_func, con_func]]
```

Reply:
Hi @Balandat , if you want I can change it accordingly, is this still possible as the PR is already imported to Phabricator?

Reply:
Yeah we can always reimport (not anymore for this one b/c it already got merged). So no worries for this one as this was really a minor nit.